tips/migrations/0002_test_data.py

```diff
@@ -0,0 +1,24 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tips', '0001_initial'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from tips.models import Tip
+
+        test_data = [
+            ('Test User1', 'A simple test message'),
+            ('Test User2', 'Another simple test message'),
```

Comment:
Can you change it to be (Test author1, some tip1)
The relation between that mock data and the Tip model is not clear